### ISSUE TITLE
Uses $Config{cpp} instead of $Config{cpprun}

### ISSUE THREAD

**Issue by kentfredric** [PerlFFI/FFI-TinyCC#11]
This is just a follow up after hitting this issue on Devel-NYTProf

https://github.com/Perl/perl5/issues/17948
https://github.com/timbunce/devel-nytprof/issues/139

When Perl is configured with `-Dcpp=$GCC -E`, the resulting `$Config{cpp}` is just `$GCC`, which is very much probably _not_ what you want. 

You probably want `$Config{cpprun}` which contains the arguments required for `$GCC` to function as a C PreProcessor.

https://github.com/PerlFFI/FFI-TinyCC/blob/e922eec5b92a934b43dfc8d9f07fdbf892687499/lib/FFI/TinyCC.pm#L318

**Comment by plicease** [comment]
`$Config{cpprun}` does seem to be more correct with `clang` as well:

```
✅ starscream% perl -V:cpp
cpp='cpp';
✅ starscream% perl -V:cpprun
cpprun='clang  -E';
```

(/usr/bin/cpp being the Gnu C preprocessor on my Debian system)

**Comment by plicease** [comment]
Thanks for the suggestion.  Updated and released as 0.30.
